webapp/scripts/functions.js

```diff
@@ -219,13 +239,21 @@ module.exports = {
    * Prints all attribute descriptions of a table
    * @param {string} table
    */
-  describeTable(mapset, table) {
-    const raw = grass(mapset, `db.describe table="${table}"`)
-    const desc = parseDescription(raw)
-    addBounds(desc, mapset, table)
-    return JSON.stringify(desc)
+  getMetadata(mapset, table) {
+    const columns = module.exports.getNumericColumns(mapset, table).map(col => col.split(':')[1].trim())
+    const data = {
+      tableObj: { headFields: ['table', 'description'], rows: [{ table: table, description: '' }] },
+      columnObj: {
+        headFields: ['column', 'description', 'min', 'max'],
+        rows: columns.map(c => { return { column: c, description: '', min: undefined, max: undefined } })
```

Comment:
equivalent to `columns.map(c => ({ column: c, description: '', min: undefined, max: undefined }))`
Also, setting undefined properties is not necessary.